java/src/hdf/hdf5lib/H5.java

```diff
@@ -10726,7 +10726,18 @@ public synchronized static native boolean H5Requal(byte[] ref1_ptr, byte[] ref2_
      * @exception IllegalArgumentException
      *                - an input array is invalid.
      **/
-    public synchronized static native long H5Ropen_object(byte[] ref_ptr, long rapl_id, long oapl_id)
+    public static long H5Ropen_object(byte[] ref_ptr, long rapl_id, long oapl_id)
```

Comment:
This is CMake related?

Reply:
No.  The fortran changes are CMake.   I think the java changes are to fix java test failures with new references.